requirements_ci.txt

```diff
@@ -0,0 +1,24 @@
+# Do not forget to add in setup.py
```

Comment:
can you explain in a comment why there is a need for a separate `requirements_ci` file?